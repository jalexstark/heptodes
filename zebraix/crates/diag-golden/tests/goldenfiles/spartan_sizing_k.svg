<?xml version="1.0" encoding="UTF-8"?>
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="400pt" height="300pt" viewBox="0 0 400 300">
<defs>
<g>
<g id="glyph-0-0">
<path d="M 0.375 -2.140625 L 0.375 -2.875 L 2.640625 -2.875 L 2.640625 -2.140625 Z M 0.375 -2.140625 "/>
</g>
<g id="glyph-0-1">
<path d="M 2.5625 -4.09375 C 3.25 -4.09375 3.796875 -3.921875 4.203125 -3.578125 C 4.609375 -3.234375 4.8125 -2.75 4.8125 -2.125 C 4.8125 -1.4375 4.59375 -0.894531 4.15625 -0.5 C 3.71875 -0.101562 3.101562 0.09375 2.3125 0.09375 C 1.96875 0.09375 1.644531 0.0625 1.34375 0 C 1.039062 -0.0625 0.789062 -0.148438 0.59375 -0.265625 L 0.59375 -1.046875 C 0.8125 -0.910156 1.082031 -0.800781 1.40625 -0.71875 C 1.726562 -0.644531 2.035156 -0.609375 2.328125 -0.609375 C 2.816406 -0.609375 3.210938 -0.722656 3.515625 -0.953125 C 3.816406 -1.191406 3.96875 -1.554688 3.96875 -2.046875 C 3.96875 -2.484375 3.832031 -2.816406 3.5625 -3.046875 C 3.300781 -3.285156 2.878906 -3.40625 2.296875 -3.40625 C 2.117188 -3.40625 1.914062 -3.390625 1.6875 -3.359375 C 1.46875 -3.328125 1.289062 -3.296875 1.15625 -3.265625 L 0.75 -3.515625 L 1 -6.671875 L 4.34375 -6.671875 L 4.34375 -5.921875 L 1.703125 -5.921875 L 1.546875 -3.984375 C 1.648438 -4.003906 1.789062 -4.023438 1.96875 -4.046875 C 2.15625 -4.078125 2.351562 -4.09375 2.5625 -4.09375 Z M 2.5625 -4.09375 "/>
</g>
<g id="glyph-0-2">
<path d="M 0.671875 -0.5 C 0.671875 -0.726562 0.726562 -0.890625 0.84375 -0.984375 C 0.957031 -1.078125 1.09375 -1.125 1.25 -1.125 C 1.40625 -1.125 1.539062 -1.078125 1.65625 -0.984375 C 1.769531 -0.890625 1.828125 -0.726562 1.828125 -0.5 C 1.828125 -0.28125 1.769531 -0.117188 1.65625 -0.015625 C 1.539062 0.078125 1.40625 0.125 1.25 0.125 C 1.09375 0.125 0.957031 0.078125 0.84375 -0.015625 C 0.726562 -0.117188 0.671875 -0.28125 0.671875 -0.5 Z M 0.671875 -0.5 "/>
</g>
<g id="glyph-0-3">
<path d="M 4.890625 -3.34375 C 4.890625 -2.625 4.8125 -2.007812 4.65625 -1.5 C 4.507812 -0.988281 4.273438 -0.59375 3.953125 -0.3125 C 3.628906 -0.0390625 3.195312 0.09375 2.65625 0.09375 C 1.90625 0.09375 1.347656 -0.207031 0.984375 -0.8125 C 0.628906 -1.425781 0.453125 -2.269531 0.453125 -3.34375 C 0.453125 -4.0625 0.523438 -4.675781 0.671875 -5.1875 C 0.816406 -5.707031 1.050781 -6.101562 1.375 -6.375 C 1.707031 -6.644531 2.132812 -6.78125 2.65625 -6.78125 C 3.40625 -6.78125 3.960938 -6.476562 4.328125 -5.875 C 4.703125 -5.269531 4.890625 -4.425781 4.890625 -3.34375 Z M 1.28125 -3.34375 C 1.28125 -2.425781 1.382812 -1.738281 1.59375 -1.28125 C 1.800781 -0.832031 2.15625 -0.609375 2.65625 -0.609375 C 3.15625 -0.609375 3.515625 -0.832031 3.734375 -1.28125 C 3.953125 -1.738281 4.0625 -2.425781 4.0625 -3.34375 C 4.0625 -4.25 3.953125 -4.929688 3.734375 -5.390625 C 3.515625 -5.847656 3.15625 -6.078125 2.65625 -6.078125 C 2.15625 -6.078125 1.800781 -5.847656 1.59375 -5.390625 C 1.382812 -4.929688 1.28125 -4.25 1.28125 -3.34375 Z M 1.28125 -3.34375 "/>
</g>
</g>
</defs>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 63.636719 286.363281 L 63.636719 13.636719 M 336.363281 286.363281 L 336.363281 13.636719 M 63.636719 286.363281 L 336.363281 286.363281 M 63.636719 13.636719 L 336.363281 13.636719 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 63.636719 150 L 336.363281 150 M 63.636719 177.273438 L 336.363281 177.273438 M 63.636719 204.546875 L 336.363281 204.546875 M 63.636719 231.816406 L 336.363281 231.816406 M 63.636719 259.089844 L 336.363281 259.089844 M 63.636719 122.726562 L 336.363281 122.726562 M 63.636719 95.453125 L 336.363281 95.453125 M 63.636719 68.183594 L 336.363281 68.183594 M 63.636719 40.910156 L 336.363281 40.910156 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 200 286.363281 L 200 13.636719 M 172.726562 286.363281 L 172.726562 13.636719 M 145.453125 286.363281 L 145.453125 13.636719 M 118.183594 286.363281 L 118.183594 13.636719 M 90.910156 286.363281 L 90.910156 13.636719 M 227.273438 286.363281 L 227.273438 13.636719 M 254.546875 286.363281 L 254.546875 13.636719 M 281.816406 286.363281 L 281.816406 13.636719 M 309.089844 286.363281 L 309.089844 13.636719 "/>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="55.636719" y="298.229492"/>
<use xlink:href="#glyph-0-1" x="58.636719" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="63.636719" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="66.636719" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="193.5" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="198.5" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="201.5" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-1" x="329.863281" y="298.229492"/>
<use xlink:href="#glyph-0-2" x="334.863281" y="298.229492"/>
<use xlink:href="#glyph-0-3" x="337.863281" y="298.229492"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-0" x="42.960938" y="289.139648"/>
<use xlink:href="#glyph-0-1" x="45.960938" y="289.139648"/>
<use xlink:href="#glyph-0-2" x="50.960938" y="289.139648"/>
<use xlink:href="#glyph-0-3" x="53.960938" y="289.139648"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-3" x="45.960938" y="152.776367"/>
<use xlink:href="#glyph-0-2" x="50.960938" y="152.776367"/>
<use xlink:href="#glyph-0-3" x="53.960938" y="152.776367"/>
</g>
<g fill="rgb(0%, 0%, 0%)" fill-opacity="1">
<use xlink:href="#glyph-0-1" x="45.960938" y="16.413086"/>
<use xlink:href="#glyph-0-2" x="50.960938" y="16.413086"/>
<use xlink:href="#glyph-0-3" x="53.960938" y="16.413086"/>
</g>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 200 150 L 200 150 M 200 150 L 200 150 "/>
<path fill="none" stroke-width="0.45" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-dasharray="4.5 3.15" stroke-miterlimit="10" d="M 200 150 L 200 150 M 200 150 L 200 150 "/>
<path fill="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke="rgb(0%, 0%, 0%)" stroke-opacity="1" stroke-miterlimit="10" d="M 150.910156 68.183594 C 150.910156 50.105469 136.257812 35.453125 118.183594 35.453125 C 100.105469 35.453125 85.453125 50.105469 85.453125 68.183594 C 85.453125 86.257812 100.105469 100.910156 118.183594 100.910156 C 136.257812 100.910156 150.910156 86.257812 150.910156 68.183594 Z M 232.726562 68.183594 C 232.726562 50.105469 218.074219 35.453125 200 35.453125 C 181.925781 35.453125 167.273438 50.105469 167.273438 68.183594 C 167.273438 86.257812 181.925781 100.910156 200 100.910156 C 218.074219 100.910156 232.726562 86.257812 232.726562 68.183594 Z M 314.546875 68.183594 C 314.546875 50.105469 299.894531 35.453125 281.816406 35.453125 C 263.742188 35.453125 249.089844 50.105469 249.089844 68.183594 C 249.089844 86.257812 263.742188 100.910156 281.816406 100.910156 C 299.894531 100.910156 314.546875 86.257812 314.546875 68.183594 Z M 314.546875 68.183594 "/>
</svg>
