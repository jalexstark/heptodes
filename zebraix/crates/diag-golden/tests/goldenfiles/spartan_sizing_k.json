{
  "sizing_scheme": "SquareCenter",
  "base_line_width": 4.0,
  "axes_range": [
    5.0
  ],
  "padding": [
    0.05
  ],
  "drawables": [
    {
      "drawable": {
        "Polyline": {
          "line_closure_choice": "Closed",
          "locations": [
            [
              -5.0,
              -5.0
            ],
            [
              -5.0,
              5.0
            ],
            [
              5.0,
              5.0
            ],
            [
              5.0,
              -5.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              -5.0,
              0.0
            ]
          ],
          "end": [
            [
              5.0,
              0.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -0.0,
              -1.0
            ],
            [
              -0.0,
              -2.0
            ],
            [
              -0.0,
              -3.0
            ],
            [
              -0.0,
              -4.0
            ],
            [
              0.0,
              1.0
            ],
            [
              0.0,
              2.0
            ],
            [
              0.0,
              3.0
            ],
            [
              0.0,
              4.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Lines": {
          "line_choice": "Light",
          "start": [
            [
              0.0,
              -5.0
            ]
          ],
          "end": [
            [
              0.0,
              5.0
            ]
          ],
          "offsets": [
            [
              0.0,
              0.0
            ],
            [
              -1.0,
              -0.0
            ],
            [
              -2.0,
              -0.0
            ],
            [
              -3.0,
              -0.0
            ],
            [
              -4.0,
              -0.0
            ],
            [
              1.0,
              0.0
            ],
            [
              2.0,
              0.0
            ],
            [
              3.0,
              0.0
            ],
            [
              4.0,
              0.0
            ]
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Both",
          "anchor_vertical": "Top",
          "texts": [
            {
              "content": "-5.0",
              "location": [
                -5.0,
                -5.0
              ]
            },
            {
              "content": "0.0",
              "location": [
                0.0,
                -5.0
              ]
            },
            {
              "content": "5.0",
              "location": [
                5.0,
                -5.0
              ]
            }
          ]
        }
      }
    },
    {
      "drawable": {
        "Text": {
          "size_choice": "Small",
          "offset_choice": "Both",
          "anchor_horizontal": "Right",
          "texts": [
            {
              "content": "-5.0",
              "location": [
                -5.0,
                -5.0
              ]
            },
            {
              "content": "0.0",
              "location": [
                -5.0,
                0.0
              ]
            },
            {
              "content": "5.0",
              "location": [
                -5.0,
                5.0
              ]
            }
          ]
        }
      }
    },
    {
      "layer": 15,
      "drawable": {
        "Circles": {
          "color_choice": "BrightRed",
          "radius": 1.2,
          "centers": [
            [
              -1.5,
              3.0
            ],
            [
              1.5,
              3.0
            ]
          ]
        }
      }
    },
    {
      "layer": 10,
      "drawable": {
        "Circles": {
          "color_choice": "Blue",
          "radius": 1.2,
          "centers": [
            [
              -3.0,
              3.0
            ],
            [
              0.0,
              3.0
            ],
            [
              3.0,
              3.0
            ]
          ]
        }
      }
    }
  ]
}
